.github/workflows/test.yml

```diff
@@ -12,7 +12,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
-          node-version: 12
+          node-version: ${{ matrix.node }}
       - run: npm test
+    strategy:
+      matrix:
+        node: [10, 12, 14, 15]
```

Comment:
could you move `strategy` above `steps`? I think that order makes more sense. Other than that this looks good 👍🏼